clarifai/client/app.py

```diff
@@ -592,6 +592,35 @@ def inputs(self,):
     """
     return Inputs.from_auth_helper(self.auth_helper)
 
+  def patch_dataset(self, dataset_id: str, action: str = 'merge', **kwargs) -> Dataset:
+    """Patches a dataset for the app.
+
+    Args:
+        dataset_id (str): The dataset ID for the dataset to create.
+        action (str): The action to perform on the dataset (merge/overwrite/remove).
+        **kwargs: Additional keyword arguments to be passed to patch the Dataset.
+
+    Returns:
+        Dataset: A Dataset object for the specified dataset ID.
+    """
+    if kwargs.get("visibility"):
+      visibility = resources_pb2.Visibility(gettable=kwargs["visibility"])
+      kwargs.update({'visibility': visibility})
+    if kwargs.get("image_url"):
+      image_pb = resources_pb2.Image(url=kwargs["image_url"])
+      kwargs.pop("image_url")
+      kwargs = {**kwargs, 'image': image_pb}
```

Comment:
You can change this block (line 609 - 612)
```
     if kwargs.get("image_url"):
       image_pb = resources_pb2.Image(url=kwargs["image_url"])
       kwargs.pop("image_url")
       kwargs = {**kwargs, 'image': image_pb}
```

to following for simplicity
```
    if "image_url" in kwargs:
      kwargs["image"] = resources_pb2.Image(url=kwargs.pop("image_url"))
```

Reply:
Changes Done